package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "faq-chatbot-edit",
+  "version": "1.0.0",
+  "description": "A chatbot using API.ai.  This is a reusable package to integrate into tools and sites where needed.",
+  "main": "scripts.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "eslint": ""
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/janie-t/FAQ-Chatbot-Package.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/janie-t/FAQ-Chatbot-Package/issues"
+  },
+  "homepage": "https://github.com/janie-t/FAQ-Chatbot-Package#readme",
+  "dependencies": {
+    "editorconfig": "^0.14.1",
+    "eslint": "^4.5.0",
+    "stylelint": "^8.0.0"
+  },
```

Comment:
These dependencies should probably be `devDependencies`

Reply:
Moved two dependencies into devDependencies.

🔥  🚒